README.md

```diff
@@ -70,88 +72,119 @@ and use the plotly.js `dist` file(s). More info [here](https://github.com/plotly
 ```
 
 ### Example illustrating use of different options together
 To create an unminified custom bundle named `myScatters` including `scatter`, `scattergl` and `scatter3d` traces without any transforms:
-```
+```sh
 npm run partial-bundle -- \
     --unminified \
     --out myScatters \
     --traces scatter,scattergl,scatter3d \
     --transforms none
 ```
 Or simply on one line:
-```
+```sh
 npm run partial-bundle -- --unminified --out myScatters --traces scatter,scattergl,scatter3d --transforms none
 ```
 
-## Building plotly.js
-
-Building instructions using `webpack`, `browserify` and other build frameworks are in [`BUILDING.md`](https://github.com/plotly/plotly.js/blob/master/BUILDING.md)
+---
+## Alternative ways to require or build plotly.js
+If your library needs to bundle or directly require [plotly.js/lib/index.js](https://github.com/plotly/plotly.js/blob/master/lib/index.js) or parts of its modules similar to [index-basic](https://github.com/plotly/plotly.js/blob/master/lib/index-basic.js) in some other way than via an official or a custom bundle, or in case you want to tweak the default build configurations of `browserify` or `webpack`, etc. then please visit [`BUILDING.md`](https://github.com/plotly/plotly.js/blob/master/BUILDING.md).
 
+---
 ## Bugs and feature requests
 
 Have a bug or a feature request? Please [open a Github issue](https://github.com/plotly/plotly.js/issues/new) keeping in mind the [issue guidelines](https://github.com/plotly/plotly.js/blob/master/.github/ISSUE_TEMPLATE.md). You may also want to read about [how changes get made to Plotly.js](https://github.com/plotly/plotly.js/blob/master/CONTRIBUTING.md)
 
+---
 ## Documentation
 
 Official plotly.js documentation is hosted at [https://plotly.com/javascript](https://plotly.com/javascript).
 
 These pages are generated by the Plotly [graphing-library-docs repo](https://github.com/plotly/graphing-library-docs) built with [Jekyll](https://jekyllrb.com/) and publicly hosted on GitHub Pages.
 For more info about contributing to Plotly documentation, please read through [contributing guidelines](https://github.com/plotly/graphing-library-docs/blob/master/README.md).
 
+### To support MathJax
+
+*Before* the plotly.js script tag, add:
+
+```html
+<script src="mathjax/MathJax.js?config=TeX-AMS-MML_SVG"></script>
+```
+
+You can get the relevant MathJax files from the internet e.g.
+"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG.js"
+
+By default, plotly.js will modify the global MathJax configuration on load.
+This can lead to undesirable behavior if plotly.js is loaded alongside
+other libraries that also rely on MathJax. To disable this global configuration
+process, set the `MathJaxConfig` property to `'local'` in the `window.PlotlyConfig`
+object.  This property must be set before the plotly.js script tag, for example:
+
+```html
+<script>
+   window.PlotlyConfig = {MathJaxConfig: 'local'}
```

Comment:
`PlotlyConfig` is no longer a thing in v2, so can't we remove all of this? Also does the script tag order even matter anymore? I bet it works now before or after, though I haven't tested this myself.

Reply:
Good call. Revisited in https://github.com/plotly/plotly.js/pull/5704/commits/5c6f724a980bbfe2ab5ba77405d9dd049fa60dad.